# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/TabParent.h
# Commit: e1fbefe38944
# Full Hash: e1fbefe38944eb82d3c2e87a0307b121daf0e830
# Author: Barret Rennie <barret@brennie.ca>
# Date: 2019-04-04 04:33:36
# Regressor Bug: 1510569
# File Overlap Count: 1
# Description:
#   Bug 1510569 - Port onProgressChange notifications inside WebProgressChild.jsm to C++ r=baku
#   
#   We do not need to handle onProgressChange64 notifications since the TabChild's
#   web progress events are filtered through an nsBrowserStatusFilter, which
#   truncates onProgresChange64 event values to 32-bit integers and then calls
# ==============================================================================

diff -r 102881458323 -r e1fbefe38944 dom/ipc/TabParent.h
--- a/dom/ipc/TabParent.h	Wed Apr 03 17:32:21 2019 +0000
+++ b/dom/ipc/TabParent.h	Wed Apr 03 17:32:41 2019 +0000
@@ -169,6 +169,12 @@
                                                       const nsString& aTitle,
                                                       nsIURI* aDocURI);
 
+  mozilla::ipc::IPCResult RecvOnProgressChange(
+      const Maybe<WebProgressData>& aWebProgressData,
+      const RequestData& aRequestData, const int32_t aCurSelfProgress,
+      const int32_t aMaxSelfProgress, const int32_t aCurTotalProgres,
+      const int32_t aMaxTotalProgress);
+
   mozilla::ipc::IPCResult RecvOnStatusChange(
       const Maybe<WebProgressData>& aWebProgressData,
       const RequestData& aRequestData, const nsresult aStatus,